_toc.yml

```diff
@@ -0,0 +1,8 @@
+format: jb-book
+root: README
```

Comment:
can we make the root page the `docs/meetup-landing-page`? Currently the first page is mostly blank

Reply:
Okay sure, will do that :+1: